locations/locations.json

```diff
@@ -2571,7 +2574,7 @@
           {
             "name": "FREE HIM",
             "item_count": 1,
-            "access_rules": [ "linen", "ruby", "throw_blood", "lorquiana","cante" ]
+            "access_rules": [ "ruby", "throw_blood", "lorquiana", "cante", "cantina", "aubade", "dawn, wheel" ]
```

Comment:
I think this means "Dawn and wheel", when they should both be "OR"

Reply:
Then the AP logic is wrong as well here, guess both should be updated